test/UnitTests/ApplicationFactoryTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using E2ETest;
+using Microsoft.Tye;
+using Microsoft.Tye.ConfigModel;
+using Tye;
+using Tye.Serialization;
+using Xunit;
+using Xunit.Abstractions;
+using YamlDotNet.RepresentationModel;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+using static E2ETest.TestHelpers;
```

Comment:
this sounds like a good use case for an additional project or a shared folder. I get kinda worried when test projects reference test projects